widgetastic_manageiq/__init__.py

```diff
@@ -1022,6 +1022,45 @@ def workaround_save_issue(self):
         )
 
 
+class ReactCodeMirror(Widget):
+    """Looks the same as ScriptBox but it is changed in 5.11
+    https://github.com/JedWatson/react-codemirror"""
+
+    ROOT = "//div[contains(@class,'miq-codemirror')]//textarea"
```

Comment:
could there be several such widgets on page ?

Reply:
I haven't seen such page. So, I saw this widget in orchestration template and I think in cloud-init. Maybe somewhere else but it was always only one field for a script.